### DISCUSSION TITLE
Is there an span.log for otlp?

### DISCUSSION THREAD

**Discussion by cg110** [jaegertracing/jaeger#4046] [Q&A] [answered]
Hi,

In the old c# jaeger client there's a span.log api for attaching logs to a span.

Having moved to opentelemetry, is there an equivalent?

I tried sending logs to the otlp end point (http/protobuf) and get a 404 not found for POSTs to v1/logs. (Which I assume is due to all-in-one not doing logs rather than the protobuf being wrong)

In some ways this doesn't surprise me in opentelemetry they're separate things, however, the old api was very useful for having the logs for a trace together with the trace.

Is there an alternative way to get logs attached to a trace when using open telemetry?

Thanks

**Comment by yurishkuro** [comment, marked as answer]
The equivalent in OTLP is span event.
